dist/templates/ovn-ipsec.yaml.j2

```diff
@@ -0,0 +1,114 @@
+---
+# ovn-ipsec
+# daemonset version 3
+# starts ipsec daemons for ovs
+# it is run on all nodes
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovn-ipsec
+  # namespace set up by install
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the ovs ipsec networking components for all nodes.
+spec:
+  selector:
+    matchLabels:
+      app: ovn-ipsec
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: ovn-ipsec
+        name: ovn-ipsec
+        component: network
+        type: infra
+        kubernetes.io/os: "linux"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: ovn
+      hostNetwork: true
+      {{ "hostPID: true" if ovn_unprivileged_mode=="no" }}
+      containers:
```

Comment:
It might be better to run the `ovs-monitor-ipsec` and `libreswan` daemons in their own containers for logging, debuggability and resource analysis reasons

Reply:
So keep it the way it is?

Thanks for the review by the way.

Reply:
Sorry if I misunderstood or misread but aren't both the `ovs-monitor-ipsec` and libreswan` processes both running in a single container called `ovn-ipsec`?

Reply:
Oh you mean run them in a separate container? BTW, have you seen this review on CNO: https://github.com/openshift/cluster-network-operator/pull/886

Reply:
Exactly, It might just be easier to track that way, and yep I just gave it a look, left a comment.  Separating these into their own containers might ease the pain of trying to handle the downstream certificate signing as well